CMakeLists.txt

```diff
@@ -67,6 +67,19 @@ foreach(p
   endif()
 endforeach()
 
+# Set old policies we have not adjusted to yet
+foreach(p
+    ## Only policies introduced after the cmake_minimum_required
+    ## version need to be set.
+
+    ##----- Policies Introduced by CMake 3.20
+    CMP0120  #: The WriteCompilerDetectionHeader module is removed.
+    )
+  if(POLICY ${p})
+    cmake_policy(SET ${p} OLD)
```

Comment:
This is a temporary solution to allow us a warning-free configuration for ITK 5.2.0, scheduled to be tagged within days (preferably tomorrow). We are tracking a proper migration in https://github.com/InsightSoftwareConsortium/ITK/issues/2465.

Reply:
Please revise the commit message and PR description to state that it is a temporary update in preparation for the release.  Also add a `FIXME` comment that explains what to do in the dev version later.

Reply:
Also then open a new issue to track setting it to `NEW` before ITK 5.3.

vxl should be fixed sooner rather than later.  If we get rid of the warning now and leave the FIXME behind, no one will ever come back and fix it later, and then some future CMake will break it when the OLD behavior is removed.